packages/dataviews/src/constants.ts

```diff
@@ -87,19 +90,19 @@ export const OPERATORS: Record< Operator, { key: string; label: string } > = {
 	},
 	[ OPERATOR_IS_ANY ]: {
 		key: 'is-any-filter',
-		label: __( 'Is any' ),
+		label: __( 'Includes' ),
```

Comment:
I wonder if all these labels should have a translators note to clarify where they are used and what the quantifiers ("any", "none", "all") refer to. From a quick look at GlotPress, I see the current translations for `es_ES`, which are grammatically inconsistent:

English | Spanish
-|-
Is none | No es ninguno
Is any | Es cualquiera